### DISCUSSION TITLE
Loading page super slow on mobile, please help, Pro support is useless

### DISCUSSION THREAD

**Discussion by hedonnn** [elementor/elementor#23026] [Developers Community]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi,

I'm hosting my website on WPX which are deemed to have fast servers. All optimization have been done: minified css & js, images optimization, CDN in place and cache.

Yet my website, and specifically my most important "money-page" fails Core Web Vitals checks: https://ibb.co/G3H6cJM 

WPX tech support had a deep look into it and came with the conclusion that:

> loaded time is coming mostly from elementor javascript files and fonts. We would recommend checking with them on the matter if there is anything that can be done to reduce the amount of such loaded fonts or files. 
> 
> Here is a screenshot as an example of what's taking the loading time -
> https://snipboard.io/gopv8i.jpg https://snipboard.io/gopv8i.jpg
> 
> as well as one from GT metrix - https://snipboard.io/DtKd6U.jpg
> https://snipboard.io/DtKd6U.jpg

I've reached out to support, as I'm a pro client of Elementor but they didn't do nothing at all and just redirected me here.

Thank you for your time and attention.

### Describe the solution you'd like

Pass the CWV checks on the GSC!

### Describe alternatives you've considered

I guess I'll use an AMP plugin and spend time and money to make sure it works well and make sure to never use Elementor again for my next project if that's the only way to go.

**Comment by m1ga** [comment]
> All optimization have been done

HTML is not minified, JS are not merged, DOM size is way too big

especially some third party plugins (royal elementor) has a very big JS file and loads multiple other JS files like particle.

So there is some parts you can still optimize. Getting a full 100 (or 99) is not impossible with Elementor, just takes some work. I use autoptimize and wp super cache most of the time. None of the bigger Elementor 3rd party addons, but many custom widgets to make the DOM as small as possible

**Comment by hedonnn** [comment]
I hired a dev since a day ago, they might have done some changes already since autooptimize was on before as wel as js minified, they might be tweaking stuff.
